wordpress-mobile/WordPress-iOS
# Remove AppBotX in favor of SKStoreReviewController

**Fixes** #6935 

Every four months or so, in the same place where we used to, under the same conditions, we will ask users how they feel about the app. If they like it we will ask the OS to trigger a review request, if they don't we will offer them to contact us via Helpshift with a specific tag: `origin:in-app-feedback`
@rachelmcr already has a filter in place to handle requests coming from here.

**To test:**

Everything has to be tested on the device since there is no App Store on the simulator.

1. On a clean install do a few interactions with notifications, and once the prompt appears say that you like the app and go thorough the flow. Please test this on 10.3 and <10.3 devices.

2. Again on a clean install do a few interactions with notifications, but this time, once the prompt appears say that you don't like it and that you want to send feedback. Check that you can send feedback and that it properly appears on Helpshift with the expected tag.

3. From Me->AppSettings->About tap on "Rate us on the App Store" and check that you are successfully redirected to write a review.

| Rate        |Give Feedback          
| ------------- |:-------------:|
|![rate](https://user-images.githubusercontent.com/5558824/27096673-31bc9244-5048-11e7-9d6c-30525f38e3c8.gif)|![feedback](https://user-images.githubusercontent.com/5558824/27096672-3182ffa2-5048-11e7-82dd-70b291128b0d.gif)|


Needs review: @astralbodies
